airflow/www/views.py

```diff
@@ -2025,6 +2028,14 @@ def clear(self):
         origin = get_safe_url(request.form.get('origin'))
         dag = current_app.dag_bag.get_dag(dag_id)
 
+        map_indexes = request.form.get('map_indexes')
```

Comment:
There’s `getlist` available for this. To use it we need to format the argument list in the front end in a specific way, but that’s better than manually parsing the list, I think.